cypress/integration/task-2.js

```diff
@@ -0,0 +1,94 @@
+describe('Task 2', function() {
+
+    it('T-2: Users should be able to log in with "User" role and see appropriate tabs', function () {
+
+        const user = "TestCon User 6"
```

Comment:
Minor: you could move this line to above the tests so all the tests could use the same variable.